reolinkapi/mixins/display.py

```diff
@@ -47,10 +47,11 @@ def set_osd(self, bg_color: bool = 0, channel: float = 0, osd_channel_enabled: b
                             "enable": osd_channel_enabled, "name": osd_channel_name,
                             "pos": osd_channel_pos
                         },
-                        "osdTime": {"enable": osd_time_enabled, "pos": osd_time_pos}
+                        "osdTime": {"enable": osd_time_enabled, "pos": osd_time_pos},
+                        "watermark": osd_watermark_enabled,
                     }}}]
         r_data = self._execute_command('SetOsd', body)[0]
-        if r_data["value"]["rspCode"] == 200:
+        if 'value' in r_data and r_data["value"]["rspCode"] == 200:
             return True
-        print("Could not set OSD. Camera responded with status:", r_data["value"])
+        print("Could not set OSD. Camera responded with status:", r_data["error"])
```

Comment:
Would this then not be the case for every mixin since they all reference the error with the "value" key?
Maybe this was recently added in a camera firmware update?

Reply:
Possibly. Haven't dove that far in. Just addressed here since that's where the handling was already.

Reply:
> Would this then not be the case for every mixin since they all reference the error with the "value" key?
> Maybe this was recently added in a camera firmware update?

Yes, it would.